osgbuild/constants.py

```diff
@@ -23,8 +23,10 @@
 KOJI_WEB = "https://koji.chtc.wisc.edu"
 
 DATA_FILE_SEARCH_PATH = [sys.path[0],
-                         os.path.join(sys.path[0], "data"),
-                         DATA_DIR]
+                         os.path.join(sys.path[0], "data")]
```

Comment:
So i think this change changes what you meant to change, but what's up with `sys.path[0]`?  What's that s'posed to be?

Reply:
The directory the script (e.g. osg-koji) is in, so you can get the data file from a local clone.

Reply:
... although it looks like it may not contain anything, it's really sys.path[0]+"/data" we want.

Reply:
or os.path.dirname(__file__)+"/data"

Reply:
>The directory the script (e.g. osg-koji) is in

Yeah i usually use `os.path.dirname(__file__)` for that (as you mention)

Reply:
OK, I updated the setting. Better now?

Reply:
I'm afraid this is worse.

`/path/to/script.py/../../data` is totally _not_ a valid path, and frankly even if `os.path.abspath` allows that i am not comfortable with it.  (Because it's wrong :-p)

`os.path.dirname(__file__) + "/../data"` is fine, or if you prefer (say because your script does a `chdir` at some point, that wrapped with `os.path.abspath(...)`